Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
+# Docker workflow for building and packaging ClusterRunner to an RPM.
+#
+
+# STAGE 1: Python-Manylinux (RHEL5) base with Python 3.4 enabled to create linux_x86_64 pex.
```

Comment:
Is Manylinux a common base image? Just curious what all the requirements are for a base image if/when we ever update to a newer version of Python.

Also, are these images pretty trustworthy? Obvs we should make sure there aren't any ways to mess with our precious clusterrunner service. :gollum_emoji:

Reply:
I'll add some comment around this. It is an official support image for building pythonw wheel.

https://www.python.org/dev/peps/pep-0513/#docker-image

Reply:
The current image has CPython 2.7, 3.3, 3.4, 3.5 and 3.6